src/Media/Media.ts

```diff
@@ -1,3 +1,10 @@
+export interface PlaceDetailMedia {
+	square?: Media;
```

Comment:
👍 
i think there will be more inconsistencies in API/interface definitions :( - will fix right away